docs/modules/ROOT/pages/how-tos/testing/build/index.adoc

```diff
@@ -1 +1,27 @@
-= Build-time tests
\ No newline at end of file
+= Build-time tests
+
+This document covers the build-time tests (formerly known as "sanity tests") that {ProductName} runs as part of its component build pipeline. These build-time tests automatically check all application images to ensure that they're up-to-date, correctly formatted, and protected from security vulnerabilities.
+
+The component build pipeline for {ProductName} include various types of build-time tests, conducted using link:https://tekton.dev/docs/pipelines/tasks/#overview[Tekton tasks]. For validating container information, the pipeline employs https://www.conftest.dev/[Conftest]. The table below outlines the currently implemented build-time tests:
```

Comment:
We are missing some tests here like clair, clamav, and the ecosystem preflight check.

Reply:
@arewm So do we want to add tables for each of these, like we have now?

- Vulnerability scanning via Clair
- Anti-virus scanning via ClamAV
- Code scanning via SAST tools

OR

Do we want to update the existing paragraph with these. For example: The pipeline employs:

- link:https://www.conftest.dev/[Conftest] to Validateg container information 
- Clair to scan vulnerabilities in the code.
- ClamAV for anti-virus scan
- Static Application Security Testing (SAST) tools to scan and analyze source code or compiled versions of code to help find security flaws

Reply:
The integration team has the [STONEINTG-766](https://issues.redhat.com/browse/STONEINTG-766) ticket to add the test information for the security checks, I think that would cover most of them, apart from the ecosystem preflight check.

Reply:
@dheerajodha @MartinBasti  fyi ^^

Reply:
Thanks for tagging me!
Yes, I plan to open a PR soon, to add details of the remaining security checks here.

Reply:
PR which addresses this comment - https://github.com/konflux-ci/docs/pull/47/files